DFC.ServiceTaxonomy.UnitTests/DFC.ServiceTaxonomy.UnitTests.csproj

```diff
@@ -14,7 +14,7 @@
     </PackageReference>
     <PackageReference Include="FakeItEasy" Version="6.2.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Does this need any bindings or do we think is all okay?

Reply:
Not 100% sure what you have in mind here, can you elaborate?